tools/image-build

```diff
@@ -33,22 +33,27 @@ image-update
 
 # ..................................................................................
 # Build base image, e.g. our version of scipy-notebook including cert handling
-image-base 
+image-base
 
 
 # ..................................................................................
 # Build STScI shared layer for all deployments
 cd ${COMMON_DIR}
 echo "========================= Building Common Image ========================"
-time docker build --tag ${COMMON_ID} --build-arg USE_FROZEN=${USE_FROZEN} .
-
+time docker build --progress plain --tag ${COMMON_ID} \
```

Comment:
Does adding `--progress` slow down the build at all?  Probably not, just wondering.

Reply:
Not that I noticed.

Reply:
We should be able to verify the timing once we get working builds again.